packages/account/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@biconomy/account",
+  "version": "2.0.0",
+  "description": "Biconomy account enable gasless transaction for any smart wallet",
```

Comment:
what this package is really doing is providing Account APIs for different contract account implementations.

grammar / description is bit off.
also use the word account or contract account instead of wallet please.